inst/rmarkdown/templates/springer/skeleton/skeleton.Rmd

```diff
@@ -1,63 +1,481 @@
 ---
-title: Title here
-subtitle: Do you have a subtitle? If so, write it here
-titlerunning: Short form of title (if too long for head)
-authorrunning: Short form of author list if too long for running head
-thanks: | 
-    Grants or other notes about the article that should go on the front 
-    page should be placed here. General acknowledgments should be placed at the
-    end of the article.
+# Supported options: 
+#   sn-nature:       Style for submissions to Nature Portfolio journals
+#   sn-basic:        Basic Springer Nature Reference Style/Chemistry Reference Style
+#   sn-mathphys:     Math and Physical Sciences Reference Style
+#   sn-aps:          American Physical Society (APS) Reference Style
+#   sn-vancouver:    Vancouver Reference Style
+#   sn-apa:          APA Reference Style 
+#   sn-chicago:      Chicago-based Humanities Reference Style
+#   default:         Default
+
+classoptions: 
+  - sn-basic      
+  - pdflatex      # Optional: Compile with pdflatex/xelatex
```

Comment:
This is a case we had in another template, and just to share that it can be done this way to automate the value based on the usual `latex_engine` configuration in the YAML 

https://github.com/rstudio/rticles/blob/ba7bb2e052bea1fdd28a8bfbb6723fde0f3307ec/R/article.R#L416-L419

https://github.com/rstudio/rticles/blob/ba7bb2e052bea1fdd28a8bfbb6723fde0f3307ec/inst/rmarkdown/templates/mdpi/resources/template.tex#L7

(note that we are missing classoption above)

Anyway, not simpler maybe, just more automated based on other configuration. Because if `latex_engine` is set to `pdflatex` (the default), or to `xelatex` this option will be required. If set to `lualatex` it needs to be removed probably. 

Just sharing in case you find that useful.